community/openwhisk/templates/invoker-pod.yaml

```diff
@@ -21,6 +21,7 @@ spec:
   selector:
     matchLabels:
       name: {{ .Values.invoker.name | quote }}
+  serviceName: {{ .Values.invoker.name | quote }}
```

Comment:
Unless there is an actual kube service with this name, I think this should be removed.  We shouldn't reference a non-existent service just to satisfy a linter message.